components/instalation/instalation.css

```diff
@@ -0,0 +1,53 @@
+@import url(/landing.css);
+@import url(/variable.css);
+
+body{
+    display: flex;
+    max-width: 100%;
+    flex-wrap: wrap;
+}
+*{
+    padding: 0;
+    margin: 0;
+    box-sizing: border-box;
+}
+
+.drawer_component{
+    display: flex;
+    /* border: 1px solid black; */
+    /* position: relative; */
+
+    
+}
+.component_list{
+    /* border: 1px solid black; */
+    display: flex;
+    gap: 1.5rem;
+    padding: 1.5rem;
+    flex-direction: column;
+    overflow-y: scroll;
+    /* overflow-x: hidden; */
+    height: 40rem;
+    width: 18rem;
+    /* background-color: red; */
+    position: sticky;
+    top: 0;
```

Comment:
As there are multiple properties in this single class, a good practice is that you can sort them in alphabetical order as that helps in debugging as well as code maintainability as suggested by Vikram Sir.

Reply:
yeah, thank you nice suggesstion.